### PR TITLE
"How do I unhide/hide game builds?"

### PR DESCRIPTION
**Pull request by indiegamehunt**

### CODE REVIEW

File: unhide-build/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## How do I unhide/hide game builds?
```
Comment (MaskofThoro) [inline unhide-build/index.md]: should we mention edit builds here as well? How do I unhide/hide/edit game builds?

Comment (indiegamehunt) [inline unhide-build/index.md]: I don't think so. It should be on the add-build page.

**Comment by indiegamehunt** [comment]
Ready for review

**Comment by indiegamehunt** [comment]
This is obsolete.

**Pull request closed without merging**